### ISSUE TITLE
Create parameter reset function for `eos.analysis` object

### ISSUE THREAD

**Issue by lorenzennio** [eos/eos#702]
Given an `eos.analysis` object, it would be useful to have a function to reset its parameters. 
@dvandyk suggested the solution:
```python
for d in eos.Parameters()
    analysis.parameters.set(d.name(), d.evaluate())
```

**Comment by dvandyk** [comment]
@lorenzennio Can we get a PR for this? :smile:

**Comment by lorenzennio** [comment]
@dvandyk sure. I am in Marseille at the moment though, so it might take a few days (:

**Comment by dvandyk** [comment]
Resolved by PR #705.
